static/app/views/issueDetails/groupActivityItem.tsx

```diff
@@ -243,14 +243,16 @@ function GroupActivityItem({activity, organization, projectId, author}: Props) {
       case GroupActivityType.SET_RESOLVED_IN_PULL_REQUEST: {
         const {data} = activity;
         const {pullRequest} = data;
-        return tct('[author] has created a PR for this issue: [version]', {
+        return tct('[author] has created a PR for this issue: [pullRequest]', {
           author,
-          version: (
+          pullRequest: pullRequest ? (
             <PullRequestLink
               inline
               pullRequest={pullRequest}
               repository={pullRequest.repository}
             />
+          ) : (
+            t('Not available')
           ),
         });
```

Comment:
The full sentence here kinda is weak:

> Ryan has created a PR for this issue: Not available

Reply:
What if it was like `t('A link to the PR is not available')` in the else case

Reply:
it took 2 years for someone to make the PR not available somehow so i'm not gonna get too crazy, just needs to not explode